HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTrackstersPassthrough_cfi.py

```diff
@@ -72,20 +60,30 @@
         algo_verbosity = cms.int32(0),
         antikt_radius = cms.double(0.09),
         computeLocalTime = cms.bool(True),
-        eid_input_name = cms.string('input'),
-        eid_min_cluster_energy = cms.double(1),
-        eid_n_clusters = cms.int32(10),
-        eid_n_layers = cms.int32(50),
-        eid_output_name_energy = cms.string('output/regressed_energy'),
-        eid_output_name_id = cms.string('output/id_probabilities'),
         minNumLayerCluster = cms.int32(5),
         type = cms.string('FastJet')
     ),
     pluginPatternRecognitionByPassthrough = cms.PSet(
         algo_verbosity = cms.int32(0),
         type = cms.string('Passthrough')
     ),
+
+    pluginInferenceAlgoTracksterInferenceByDNN = cms.PSet(
+      algo_verbosity = cms.int32(0),
+      onnxPIDModelPath = cms.FileInPath('RecoHGCal/TICL/data/RecoHGCal-TICL/ticlv4/onnx_models/energy_id_v0.onnx'),
+      onnxEnergyModelPath = cms.FileInPath('RecoHGCal/TICL/data/RecoHGCal-TICL/ticlv4/onnx_models/energy_id_v0.onnx'),
```

Comment:
These 2 are not used in TICLv4, but for coherency, these 2 should be pointing to ticlv5 models (PassThrough pattern recognition is only used in TICLv5)